src/styles/previewStyles.less

```diff
@@ -109,6 +110,12 @@
 					}
 				}
 
+				#pageRangeControl {
+					// Temporary workaround that fixes styling issues with the PDF Preview Header
```

Comment:
The 300px is large enough so that the width of the input as well as the radio buttons + labels all fit in the same box. Matthew brought up the "real" fix, which is some underlying differences in how Safari and Chrome handle CSS. I will create an issue for the real fix. I'd like to get this out the door